### DISCUSSION TITLE
What about cusmtom statuses of entries?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [craftcms/cms#12700] [Ideas] [answered]
Just like commerce plugin does, would be great t ohave it for entries also, as default...

**Comment by brandonkelly** [comment, marked as answer]
Commerce order statuses are 100% user-defined, and don’t actually mean anything to Commerce (beyond whatever custom behaviors have been assigned to them by the developer).

Entry statuses on the other hand are actually meaningful to Craft, and identify the actual entry state. So I’m not sure how custom statuses would play into that.
